empire/server/modules/python/discovery/nameserver.yaml

```diff
@@ -0,0 +1,23 @@
+name: Nameserver IP
+authors:
+  - name: '0x636f646f'
```

Comment:
Noticed a bug in the GitHub install method, the name is being interpreted as int data. Adding single quotes to convert to string:

```
[INFO]: v2: Loading modules from: /home/ubuntu/development/git/Empire/empire/server/modules/ 
[ERROR]: Error loading module nameserver.yaml: 1 validation error for EmpireModule
authors.0.name
  Input should be a valid string [type=string_type, input_value=1668244591, input_type=int]
    For further information visit https://errors.pydantic.dev/2.8/v/string_type 

```